Sources/ClientRuntime/Config/DefaultSDKRuntimeConfiguration.swift

```diff
@@ -15,10 +15,12 @@ public struct DefaultSDKRuntimeConfiguration: SDKRuntimeConfiguration {
     public let retryer: SDKRetryer
     public var clientLogMode: ClientLogMode
     public var endpoint: String?
+    public var partitionID: String?
```

Comment:
Can we add the docs from the protocol here as well?

Reply:
Added